services/packages/server/packages.go

```diff
@@ -161,6 +164,23 @@ var (
 		return genCmd(p, listOpts)
 	}
 
+	generateSearch = func(p *pb.SearchRequest, searchInstalled bool) ([]string, error) {
+		var out []string
+		switch p.PackageSystem {
+		case pb.PackageSystem_PACKAGE_SYSTEM_YUM:
+			out = append(out, RepoqueryBin)
+			out = append(out, "--nevra")
+			out = append(out, p.Name)
+			if searchInstalled {
+				out = append(out, "--installed")
```

Comment:
what's the default behavior if there's no `--installed`?

Reply:
Based on https://man7.org/linux/man-pages/man1/repoquery.1.html, repoquery firefox = repoquery -a(-all) firefox  which will display the latest version of that package, but if we set --installed, then the -all wouldn't be set

Reply:
I can add -all to reduce confusion if we want to get latest version